pkg/runtime/config.go

```diff
@@ -93,7 +93,7 @@ type Config struct {
 	DaprBlockShutdownDuration    *time.Duration
 	ActorsService                string
 	RemindersService             string
-	SchedulerServiceHostAddr     string
+	SchedulerHostAddr            string
```

Comment:
```suggestion
	SchedulerHostAddr            *string
```

Reply:
if thats ok with you, I'd like to leave this as is and merge this change in in the [open PR here](https://github.com/cicoyle/dapr/pull/12/files#diff-445f644b5a98369d907635a0901d9922efd1d0f6470ab80696d12ef46a4c7525)